js/noms/src/remote-batch-store.js

```diff
@@ -156,11 +156,13 @@ export function makeTestingRemoteBatchStore(): BatchStore {
   return new RemoteBatchStore(3, new TestingDelegate(new MemoryStore()));
 }
 
-class TestingDelegate {
+export class TestingDelegate {
   _cs: ChunkStore;
+  preUpdateRootHook: () => Promise<void>;
 
   constructor(cs: ChunkStore) {
     this._cs = cs;
+    this.preUpdateRootHook = () => Promise.resolve();
```

Comment:
It seems like this is probably the wrong no-op function to put here, but I couldn't get anything with different syntax to work.